Classes/Controller/AdministrationController.php

```diff
@@ -289,7 +289,7 @@ public function indexAction()
         $dblist->script = GeneralUtility::getIndpEnv('REQUEST_URI');
         $dblist->thumbs = $this->getBackendUser()->uc['thumbnailsByDefault'];
         $dblist->allFields = 1;
-        $dblist->localizationView = 1;
+        $dblist->localizationView = $this->tsConfiguration['localizationView'] ? MathUtility::forceIntegerInRange($this->tsConfiguration['localizationView'], 0) : 1;
```

Comment:
just do a type cast to int, no need for usage of MathUtility

Reply:
If i set (int) the default value for non-configured settings is 0, and that was a changed to the given behaviour. If you are agreed with this, i will change to integer.